features/support/pages/guidance_page.rb

```diff
@@ -0,0 +1,6 @@
+class GuidancePage < SitePrism::Page
+  set_url '/guidance/new'
+
+  element :name, 'input.t-name'
+  element :submit, 'input[type="submit"]'
```

Comment:
Sure, I've no argument not to use CSS classes to target all the elements on the page